### PR TITLE
updated documentation to include User Story and API readiness checklist

### PR DESCRIPTION
**Pull request by maheshc01**
#### What type of PR is this?

Add one of the following kinds:
* bug
* correction
* enhancement/feature
* cleanup
* documentation
included API readiness checklist and user story
* subproject management
* tests

#### Which issue(s) this PR fixes:

Fixes #

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Comment by maheshc01** [comment]
included another commit to the PR which includes additional documentation as part of the yaml file

File: code/API_definitions/CAMARA Connectivity Insights API.yaml [hunk 1]
```diff
@@ -1,8 +1,37 @@
 ---
     openapi: 3.0.3
     info:
-      title: Connectivity Insights API
-      description: Find out the likelihood of the network meeting the networking requirements of your application for a given user session,.
+      title: Connectivity Insights
+      description: |
+        Connectivity Insights API provides programmable interface for developers and other users (capabilities consumers) with the much needed visibility into the network performance data in the form of an easy to understand and use intent based API.
+        <br>This empowers the developers to take critical decisions to ensure their users are able to get the best experience while using the applciation.
```
Comment (Kevsy) [inline code/API_definitions/CAMARA Connectivity Insights API.yaml:7]: s/applciation/application

File: documentation/API_documentation/ConnectivityInsights-User-Story.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Connectivity Insights API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an application developer i need to be able to configure what my application requires in terms of network perfomance metrics and be able to get visibility into actual network performance as experienced by my customers in comparison to the expected thresholds configured. In addtion i would also like to get notifications when the network is not able to meet the thresholds configured for indivial customers.  |
```
Comment (Kevsy) [inline documentation/API_documentation/ConnectivityInsights-User-Story.md:5]: s/i/I

Comment (Kevsy) [inline documentation/API_documentation/ConnectivityInsights-User-Story.md:5]: s/indivial/individual

Comment (Kevsy) [inline documentation/API_documentation/ConnectivityInsights-User-Story.md:5]: I think 'configure' may be misinterpreted , as it makes it sound like an programming action that chnages network treatment (like QoD). 

As an application developer I need to be able to understand what my application requires in terms of network performance demands,  and be able to determine the likelihood that the network will be able to deliver performance that meets those demands.  I can represent my application's network performance demands as a set of metrics, and register these with the network. At runtime I can query the network to determine if a given customer device is likely to receive that level of network performance across each metric.  In addition I would also like to get notifications when the network is not able to meet the network performance thresholds for given customers.

File: documentation/API_documentation/SequenceDiagram.png
Comment (Kevsy) [inline documentation/API_documentation/SequenceDiagram.png:1]: Should that be 'Subscription ID' at the second-to- bottom flow?

**Review by Kevsy — CHANGES_REQUESTED** [review]
See comments inline

File: code/API_definitions/CAMARA Connectivity Insights API.yaml [hunk 3]
```diff
@@ -2,13 +2,42 @@
...
       title: Connectivity Insights API
-      description: Find out the likelihood of the network meeting the networking requirements of your application for a given user session,.
       version: '0.3.0-wip'
       contact: 
        email: sp-coi@lists.camaraproject.org
       license:
         name: Apache 2.0
         url: https://www.apache.org/licenses/LICENSE-2.0.html
+      description: |
+        Connectivity Insights API provides programmable interface for developers and other users (capabilities consumers) with the much needed visibility into the network performance data in the form of an easy to understand and use intent based API.
```
Comment (Kevsy) [inline code/API_definitions/CAMARA Connectivity Insights API.yaml:12]: I think the first two paragraphs are too 'marketing-speak'. How about the paragraph I suggested followed by key points from  your 2nd and 3rd paragraph, i.e.:
"
The Connectivity Insights API allows an application developer to ask the network the likelihood that an  application's networking requirements can be met for a given end user session.

This information helps a developer  ensure their end users are able to get the best experience while using the application over their current network. 
        Depending on the answer the network gives, the developer may decide to request a network boost (via the CAMARA QoD API) , and/or apply specific changes on the application side e.g. adjusting the resolution of the video stream upwards or downwards."

**Review by Kevsy — CHANGES_REQUESTED** [review]
See inline comment for API Description in the YAML file for suggested change.

File: code/API_definitions/CAMARA Connectivity Insights API.yaml (hunk 3 above)
Comment (maheshc01) [reply code/API_definitions/CAMARA Connectivity Insights API.yaml:12]: incorporated this feedback and by incorporating the suggested text.

**Comment by maheshc01** [comment]
@Kevsy have incorporated the feedback suggested. Request to review and approve the pull request.

**Review by Kevsy — APPROVED** [review]

**Pull request merged**